### ISSUE TITLE
A player blatantly smurfing after new report system update. 

### ISSUE THREAD

**Issue by Necrofastz** [ValveSoftware/Dota2-Gameplay#11841]
### Description

A player blatantly smurfing after new report system update. Ruining lower ranked immortal games.

Evidence:
His Dotabuff : https://www.dotabuff.com/players/1549051874 

### Screenshots

![Dota 2 9_11_2023 12_36_39 PM](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143851968/ebbd6d48-9a5c-4891-8560-333401c27e57)

**Comment by TobyFlendersonFromHR** [comment]
thats not even smurfing

**Comment by prohjort** [comment]
> thats not even smurfing

yeah, he's just a talented player who got to 1k immortal after 700 games.
